frontend/packages/dev-console/src/components/helm/HelmActionForm.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
```

Comment:
`HelmActionForm` might be misleading here since we have a lot more Helm actions than install and upgrade. Probably better to name it `HelmInstallUpgradeForm`. WDYT?

Reply:
Ignore this. It would create another confusion. We already have `HelmInstallPage` and `HelmUpgradePage`.

Reply:
Removed `HelmInstallPag` & `HelmReleaseUpgradePage` and have created one common `HelmInstallUpgradePage`